var/spack/repos/builtin/packages/modeltest-ng/package.py

```diff
@@ -24,5 +25,9 @@ class ModeltestNg(CMakePackage):
     depends_on("flex", type="build")
     depends_on("openmpi", when="+mpi")
 
```

Comment:
Please add a conflict for the ICE with gcc@12.2.1 on aarch64 (see my comment):
```suggestion
   # 40217: ICE by gcc-toolset-12-gcc-12.2.1-7.4.el8.aarch64 of Rocky Linux 8.8:
   conflicts("%gcc@12.2.1", when="target=aarch64:")
```

Reply:
Was able to reproduce this on alma. I don't readily have a real rhel box or any other distros installed on aarch64 to see if it's just the rebuilds or only rhel, but I'm fine with this.